Modules/Quotas/src/Quotas.ConsumerApi/Controllers/QuotasController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Backbone.BuildingBlocks.API;
+using Backbone.BuildingBlocks.API.Mvc;
+using Backbone.BuildingBlocks.API.Mvc.ControllerAttributes;
+using Backbone.Modules.Quotas.Application.DTOs;
+using Backbone.Modules.Quotas.Application.Identities.Queries.ListQuotasForIdentity;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Backbone.Modules.Quotas.ConsumerApi.Controllers;
+
+[Route("api/v1/[controller]")]
+[Authorize("OpenIddict.Validation.AspNetCore")]
+public class QuotasController : ApiControllerBase
+{
+    public QuotasController(IMediator mediator) : base(mediator) { }
+
+    [HttpGet()]
+    [ProducesResponseType(typeof(HttpResponseEnvelopeResult<List<QuotaDTO>>), StatusCodes.Status200OK)]
+    [ProducesError(StatusCodes.Status400BadRequest)]
```

Comment:
In which case will there be a 400?

Reply:
Removed, in no case should it return an error.